torchvision/prototype/features/_label.py

```diff
@@ -26,7 +26,7 @@ def __new__(
 
     @classmethod
     def from_category(cls, category: str, *, categories: Sequence[str]):
-        categories = list(categories)
+        categories = list(categories)  # why shallow copy here? If this method is in a loop, we run the risk of creating many shallow-copies
```

Comment:
When writing that, I wasn't sure all `Sequence`'s supported the `.index()` method, but I forgot to check. Since [that seems to be the case](https://docs.python.org/3/library/collections.abc.html?highlight=sequence#collections.abc.Sequence), there is no need for the conversion to a list.